tools/legal-review/Standard/org.xerial.sqlite-jdbc-3.34.0/copyright-keep

```diff
@@ -0,0 +1,8 @@
+Copyright (c) 2007 David Crawshaw <david@zentus.com>
+Copyright 2007 Taro L. Saito
+Copyright 2008 Taro L. Saito
+Copyright 2009 Taro L. Saito
+Copyright 2010 Taro L. Saito
+Copyright 2016 Magnus Reftel
+copyright notice and this permission notice appear in all copies.
+this work for additional information regarding copyright ownership.
```

Comment:
These are just some fragments, not the whole thing.

If they are only redundant - they should be ignored in my opinion.
Otherwise, if it is possible, they should be kept as `Keep With Context` to include the whole context. That may not always be possible - in such case they should be ignored **and** their content should be added manually by pasting it into a file called `copyright-add` placed next to this file.